canonical-web-and-design/ubuntu.com
# Add raspberry-pi to the footer, add redirect

## Done

- Add raspberry-pi to the footer
- Add redirect for /raspberrypi/*
- Remove old security constraint from footer item code

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/raspberrypi
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are redirected, see that the Raspberry pi section is in the footer, see that Security section is still in the footer


## Screenshots

![image](https://user-images.githubusercontent.com/441217/96843406-2f5fb600-1446-11eb-960a-cf15d184efb0.png)
